### PR TITLE
Add tag updates

### PR DESCRIPTION
**Pull request by Robinlovelace**

### CODE REVIEW

**Comment by Robinlovelace** [comment]
Great to see these changes rolling in again @AnnaGoodman1, thanks for switching branches. Will hold my horses on review until I get a notification.

**Comment by AnnaGoodman1** [comment]
Ready for code review and merging....

**Comment by AnnaGoodman1** [comment]
@nikolai-b , do you want to review this before it is merged? If so could you do in next week or so - it is meant to be live by end November

File: regions_www/m/pct_shiny_funs.R [hunk 1]
```diff
@@ -589,8 +607,14 @@ popup_zones <- function(data, scenario, purpose){
     </tr>
     <tr>
       <td> Change in deaths/yr: &nbsp; </td>
-      <td style= 'color:", font_colour , "' >", round(data[[data_filter(scenario, "sideath_heat")]], 3),
-             " (&pound;" , round(data[[data_filter(scenario, "sivalue_heat")]]), ")
+        <td style= 'color:", font_colour_death , "' >", round(data[[data_filter(scenario, "sideath")]], 3),
```
Comment (nikolai-b) [inline regions_www/m/pct_shiny_funs.R]: We are using the colour based on `sivalueyll` but we are displaying `sideath` (and `sivalueyll`).  I'm guessing they are pretty much the same but wanted to check.

File: regions_www/tabs/manual_body.html [hunk 2]
```diff
@@ -157,6 +157,18 @@ <h4><a name="faqs"></a>Additional FAQs</h4>
...
+             <p>The PCT presents results in terms of numbers of people travelling to work or school in different ways – e.g. the number of commuters cycling to work in different scenarios.  This is in line with the input data we are using from the Census (for commuting) and the National School Census (for school travel).</p>
+             <p>We appreciate some users will be more interested in numbers of trips than numbers of people. We recommend estimating numbers of trips affected as: <b>Change in number of people * mean cycling trips per cyclist per week * 52.2 weeks in the year</b></p>
+             <p>The change in the number of people (e.g. an increase in cyclists or a decrease in drivers) is given in the PCT outputs. We estimate mean number of cycling trips per cyclist per week from the National Travel Survey (2010-2016) as:<ul>
+<li>5.1 for commuters who say cycling is their usual mode of travelling to school</li>
+<li>2.3 for primary school children whose parents say cycling is their usual mode of travelling to school</li>
+<li>5.1 for secondary school children whose parents say cycling is their usual mode of travelling to school</li></ul></p>
```
Comment (nikolai-b) [inline regions_www/tabs/manual_body.html]: This `</p>` isn't valid.  It looks ok but `<p>` tags can't contain `<ul>` tags (see [HTML spec](https://html.spec.whatwg.org/dev/grouping-content.html#the-p-element)) so it should be:

```html
<p>
  the paragraph text
</p>
<ul>
  <li>Item 1</li>
  <li>Item 2</li>
</ul>
```

File: regions_www/tabs/manual_body.html [hunk 3]
```diff
@@ -157,6 +157,18 @@ <h4><a name="faqs"></a>Additional FAQs</h4>
                 </table>
               </div>
             </div>
+            <p><a href="javascript:toggleElement(&#39;a4&#39;)">4. What if I'm interested in trips not people? And does PCT account for return trips?</a></p>
+             <div id="a4" style="display:none">
+             <p>The PCT presents results in terms of numbers of people travelling to work or school in different ways – e.g. the number of commuters cycling to work in different scenarios.  This is in line with the input data we are using from the Census (for commuting) and the National School Census (for school travel).</p>
+             <p>We appreciate some users will be more interested in numbers of trips than numbers of people. We recommend estimating numbers of trips affected as: <b>Change in number of people * mean cycling trips per cyclist per week * 52.2 weeks in the year</b></p>
+             <p>The change in the number of people (e.g. an increase in cyclists or a decrease in drivers) is given in the PCT outputs. We estimate mean number of cycling trips per cyclist per week from the National Travel Survey (2010-2016) as:<ul>
```
Comment (nikolai-b) [inline regions_www/tabs/manual_body.html]: Minor but is it worth a link to https://www.gov.uk/government/collections/national-travel-survey-statistics ?

File: regions_www/tabs/manual_body.html [hunk 4]
```diff
@@ -157,6 +157,18 @@ <h4><a name="faqs"></a>Additional FAQs</h4>
...
+<li>5.1 for commuters who say cycling is their usual mode of travelling to school</li>
+<li>2.3 for primary school children whose parents say cycling is their usual mode of travelling to school</li>
+<li>5.1 for secondary school children whose parents say cycling is their usual mode of travelling to school</li></ul></p>
+             <p>Note that all these values are much less than 10, i.e. one cannot assume all commuter and schoolchild cyclists make five return trips per week across the year.  This is because of factors including part-time working; holiday periods; and cyclists using other modes for some journeys.</p>
+             <p>We have ourselves used a very similar method for the purposes of calculating physical activity benefits and carbon emission savings - see User Manuals C1 and C2. The only difference is that for the commuting layer we do not use the total population value '5.1' but instead used equivalent values calculated separately by age and gender (range 4.1-5.5).</p>
```
Comment (nikolai-b) [inline regions_www/tabs/manual_body.html]: Is it worth a link to the User Manuals C1 and C2?

File: regions_www/tabs/manual_body.html [hunk 5]
```diff
@@ -157,6 +157,18 @@ <h4><a name="faqs"></a>Additional FAQs</h4>
                 </table>
               </div>
             </div>
+            <p><a href="javascript:toggleElement(&#39;a4&#39;)">4. What if I'm interested in trips not people? And does PCT account for return trips?</a></p>
```
Comment (nikolai-b) [inline regions_www/tabs/manual_body.html]: Wow, is that how we do things in the manual? Hope it wasn't hard to work out, `href="javascript:toggleElement(&#39;a4&#39;)"` looks a touch esoteric.

File: outputs_regional_sha [hunk 6]
```diff
@@ -1 +1 @@
-1d59549
\ No newline at end of file
+f8576bc
```
[the remark this answers is not in the document]
Comment (AnnaGoodman1) [reply outputs_regional_sha]: I think it is correct, this now points to Rsmall repo

File: regions_www/tabs/manual_body.html (hunk 5 above)
Comment (AnnaGoodman1) [reply regions_www/tabs/manual_body.html]: Agreed esoteric-looking, I just copied it from above...

File: regions_www/tabs/manual_body.html (hunk 2 above)
Comment (AnnaGoodman1) [reply regions_www/tabs/manual_body.html]: fixed, thanks

File: regions_www/tabs/manual_body.html (hunk 3 above)
Comment (AnnaGoodman1) [reply regions_www/tabs/manual_body.html]: Good thought to think I would rather not, as we don't actually get the data from there but from one of their data releases... And also government URLs are always changing

File: regions_www/tabs/manual_body.html (hunk 4 above)
Comment (AnnaGoodman1) [reply regions_www/tabs/manual_body.html]: I think better not as they exist in two versions – English and Welsh. plus they are further up the same page.

File: regions_www/m/pct_shiny_funs.R (hunk 1 above)
Comment (AnnaGoodman1) [reply regions_www/m/pct_shiny_funs.R]: yes thanks, when one is negative the other is almost always positive...and in the 0.2% where that is not true (and both are v close to zero) makes sense to me to choose just one colour, and on balance the sivalueyll is the better one.

Comment (AnnaGoodman1) [reply regions_www/m/pct_shiny_funs.R]: The reason they could be different is that deaths are weighted equally, whereas (YLLs are weighted according to the age of the person. So if you got three extra deaths among older people but two were averted among younger people then you could possibly get the two numbers pointing in different directions. It is rare)

**Comment by AnnaGoodman1** [comment]
thanks so much @nikolai-b . Happy for me to merge this to test server now?

File: regions_www/tabs/manual_body.html (hunk 5 above)
Comment (nikolai-b) [reply regions_www/tabs/manual_body.html]: Took me bit to see what it did, I guess it was auto-generated from Rmarkdown once.  There are many ways to improve to make it simpler and easier to understand.  The standard way to use [data attributes](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes), then write similar Javascript code to open and close based on the attributes BUT I'm not sure it is worth the effort for the manual and it does work :smile:

File: outputs_regional_sha (hunk 6 above)
[the remark this answers is not in the document]
Comment (nikolai-b) [reply outputs_regional_sha]: Ah, thank you very much for reminding me.  Maybe we should make that much clearer somehow as I'd forgotten.  Just changing the file name to `pct-outputs-regional-Rsmall` might be enough.

[the remark this answers is not in the document]
Comment (AnnaGoodman1) [reply outputs_regional_sha]: ok did search and replace in pct-shiny changing name, will assume caught all places. and merge...but maybe just check @nikolai-b ?

**Pull request merged**